File: FunStory/app/src/main/java/com/css/funstory/MainActivity.java
package com.css.funstory;

import android.support.constraint.ConstraintLayout;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.text.Layout;
import android.util.Log;
import android.view.LayoutInflater;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import com.css.funstory.bussiness.Book;
import com.css.funstory.bussiness.ColorWheel;
import com.css.funstory.entity.FunStory;

import org.w3c.dom.Text;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button button = findViewById(R.id.button);
        changeUI();
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                changeUI();
            }
        });
    }

    public void changeUI() {
        TextView tv1 = findViewById(R.id.title);
        TextView tv2 = findViewById(R.id.content);
        FunStory funStory = new Book().getFunStory();
        tv1.setText(funStory.getTitle());
        tv2.setText(funStory.getContent());
        ConstraintLayout layout = findViewById(R.id.layout);
        layout.setBackgroundColor(new ColorWheel().getColor());
    }
}

File: Project/app/src/main/java/com/css/project/MainActivity.kt
package com.css.project

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.text.InputType
import kotlinx.android.synthetic.main.home.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val projectGroup = ProjectGroup("剪纸堆",R.drawable.image,
            "陈姗姗，黄鲁菲，汤瑶，金悦","""
               功能列表
                1、保存复制的文字，生成一条剪贴板记录
                2、编辑（暂时不包括新建）剪切板记录
                3、删除选中的剪切板记录，
                    （1）逐条删除
                    （2）多选删除
                    （3）清空所有
                4、合并选中的多条剪切记录
                5、给某些剪切记录加星收藏
                6、分享剪切记录
            """, "小仙女")
        tv1.text = projectGroup.projectName
        image1.setImageResource(projectGroup.projectIcon)
        tv4.text = projectGroup.projectDesc
        tv2.text = projectGroup.projectNumber
        btn.setOnClickListener{
            tv3.text = et.text
//            et.inputType = InputType.TYPE_CLASS_TEXT
        }
    }
}
